articles/client-platforms/angular2/06-rules.md

```diff
@@ -33,4 +32,4 @@ Then, display user profile's new `country` attribute added in the created rule a
 ![Country rule sample](/media/articles/angularjs2/rule-country-show.png)
```

Comment:
the file is working in prod: https://auth0.com/docs/quickstart/spa/angular2/06-rules
I think images are not working in your test env, cause I don't see any image there..